docs/ansible_runner_integration.md

```diff
@@ -1,19 +1,19 @@
 ## Ansible Runner Integration Overview
 
-Much of the code in AWX around ansible and ansible-playbook invocation interacting has been removed and put into the project ansible-runner. AWX now calls out to ansible-runner to invoke ansible and ansible-playbook.
+Much of the code in AWX around Ansible and `ansible-playbook` invocation has been removed and put into the project `ansible-runner`. AWX now calls out to `ansible-runner` to invoke Ansible and `ansible-playbook`.
```

Comment:
These should probably be `ansible`, since that's the binary used for adhoc commands, which I think is the intention here.

Reply:
Makes sense!